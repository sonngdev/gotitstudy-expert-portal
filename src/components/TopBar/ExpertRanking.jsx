import React from 'react';

export function ExpertRanking() {
  return (
    <div className="expert-ranking u-backgroundWhite u-roundedLarge u-paddingTiny u-paddingHorizontalExtraSmall u-text100 u-textUppercase u-textPrimaryDark">
      Your ranking
      <div className="stat u-text600 u-textCenter">0</div>
    </div>
  )
}
