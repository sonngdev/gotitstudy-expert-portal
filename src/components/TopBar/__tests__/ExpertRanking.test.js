import React from 'react';
import { shallow } from 'enzyme';
import { ExpertRanking } from '../ExpertRanking';

describe('<ExpertRanking />', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<ExpertRanking />);
  });

  it('renders', () => {
    expect(wrapper.debug()).toMatchSnapshot();
  });

  it('contains static ranking for now', () => {
    const ranking = wrapper.find('.stat');
    expect(ranking.text()).toBe('0');
  });
});
