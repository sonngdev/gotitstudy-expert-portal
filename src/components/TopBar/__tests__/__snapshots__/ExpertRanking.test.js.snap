// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ExpertRanking /> renders 1`] = `
"<div className=\\"expert-ranking u-backgroundWhite u-roundedLarge u-paddingTiny u-paddingHorizontalExtraSmall u-text100 u-textUppercase u-textPrimaryDark\\">
  Your ranking
  <div className=\\"stat u-text600 u-textCenter\\">
    0
  </div>
</div>"
`;
